tests/test_utils.py

```diff
@@ -160,3 +161,45 @@ def test_get_option_value():
     assert get_option_value(option_value_no_label) == 'somevalue'
     assert get_option_value(option_label_no_value) == 'somelabel'
     assert get_option_value(option_label_and_value) == 'somevalue'
+
+
+class TestTryLoadManifest:
+    FRAMEWORK_DATA = {'slug': 'framework'}
+    QUESTION_SET = 'services'
+    MANIFEST = 'example'
+
+    def setup(self):
+        self.content_loader_mock = mock.Mock()
+        self.application_mock = mock.Mock()
+
+    def test_content_loader_asked_to_load_manifest(self):
+        try_load_manifest(self.content_loader_mock, self.application_mock,
+                          TestTryLoadManifest.FRAMEWORK_DATA,
+                          TestTryLoadManifest.QUESTION_SET,
+                          TestTryLoadManifest.MANIFEST
+                          )
+
+        assert self.content_loader_mock.load_manifest.called is True
+        self.content_loader_mock.load_manifest.assert_called_with(
```

Comment:
Should this maybe be `assert_called_once_with`?

Or even (I know @katstevens prefers this way) 
```
self.content_loader_mock.load_manifest.call_args_list == [
    mock.call(TestTryLoadManifest.FRAMEWORK_DATA['slug'], 
              TestTryLoadManifest.QUESTION_SET,
              TestTryLoadManifest.MANIFEST)
]
```

Reply:
I'm OK with the latter way. I'll tweak.